Content.Server/_EE/FootPrint/FootPrintsSystem.cs

```diff
@@ -21,6 +25,16 @@ public sealed class FootPrintsSystem : EntitySystem
     [Dependency] private readonly SharedSolutionContainerSystem _solution = default!;
     [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
     [Dependency] private readonly SharedTransformSystem _transform = default!;
+    [Dependency] private readonly SharedMapSystem _mapSystem = default!; // DeltaV
+
+    // DeltaV - Max amount of footprints per tile
+    // Not stored on the component because its convenient here
+    private const int MaxFootprintsPerTile = 2;
+
+    /// <summary>
+    ///     DeltaV: Dictionary tracking footprints per tile using tile coordinates as key
+    /// </summary>
+    private readonly Dictionary<Vector2i, Queue<EntityUid>> _footprintsPerTile = new();
 
```

Comment:
If you don't store this in a component it gets nuked on any downstreams trying persistence.

Reply:
fuck you're right, i forget that persistence is even a thing